JS-test/main.js

```diff
@@ -0,0 +1,34 @@
+//Write a function that logs to the console numbers 1 to 100.
+function counting() {
+  for (let i = 1; i < 101; i++) {
+    if (i % 3 == 0 && i % 5 == 0) {
+      console.log("Jackpot!");
+    } else if (i % 3 == 0) {
+      console.log(" This is divisible by 3");
+    } else if (i % 5 == 0) {
+      console.log("This is divisible by 5");
+    } else console.log(i);
```

Comment:
```suggestion
    } else {
      console.log(i);
    }
```

Prefer to be consistent with your syntax, if you use `{}` in part of the `if` use it in all parts of the `if`, and generally I would advice to always use both `{}` and `;` even if they are optional in many places in JS.